### PR TITLE
fixes all but one of the test errors for collection type

### PR DESCRIPTION
**Pull request by sdellis**
@samvera/hyrax-code-reviewers

### CODE REVIEW

**Comment by no-reply** [comment]
@sdellis it seems like the build is failing due to `NameError: uninitialized constant Hydra::RoleManagement`, can you address that?

**Comment by no-reply** [comment]
i think if the `Hydra::RoleManagement::UserRoles` changes are removed, and this is rebased on `main` it will pass again

**Comment by sdellis** [comment]
@no-reply I reverted the Dassie User model, rebased to main, and pushed it.

File: .koppie/app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 class User < ApplicationRecord
+  # Connects this user object to Role-management behaviors.
+  include Hydra::RoleManagement::UserRoles
```
Comment (hackartisan) [inline .koppie/app/models/user.rb:3]: does this need to be added in `lib/generators/hyrax/models_generator.rb`?

Comment (hackartisan) [reply .koppie/app/models/user.rb:3]: Oh, this was removed in https://github.com/samvera/hyrax/pull/6272/files. Should it be added back in here?

Comment (no-reply) [reply .koppie/app/models/user.rb:3]: it shouldn't, i think

**Review by hackartisan — APPROVED** [review]

**Pull request merged**